### PR TITLE
Add Google play link of the App to Readme file

### PR DESCRIPTION
**Pull request by Ahmed9914**
السلام عليكم
This PR will fix future issues same like issue #983.

### CODE REVIEW

**Comment by obayit** [comment]
How this is going to fix #983?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-[![Build Status](https://travis-ci.org/quran/quran_android.svg?branch=master)](https://travis-ci.org/quran/quran_android)
-
-# Quran for Android
+[![Build Status](https://travis-ci.org/quran/quran_android.svg?branch=master)](https://travis-ci.org/quran/quran_android)   
 
+[![Get it on Google Play](http://i.imgur.com/7sq06lr.png)](https://play.google.com/store/apps/details?id=com.quran.labs.androidquran)
```
Comment (ahmedre) [inline README.md]: can we use this instead - https://play.google.com/intl/en_us/badges/

**Comment by ahmedre** [comment]
@Ahmed9914 can we make the button smaller please?

**Comment by ozbek** [comment]
Actually, the app logo was put in README just for that purpose. I will replace the app logo with the Play Store badge...

**Comment by ozbek** [comment]
> How this is going to fix #983?

Perhaps, @Ahmed9914 meant my comment asking you if you are using the correct app link.

**Pull request merged**